tony-core/src/main/java/com/linkedin/tony/TonyClient.java

```diff
@@ -930,6 +930,12 @@ String buildCommand(long amMemory) {
         + ApplicationConstants.LOG_DIR_EXPANSION_VAR);
     // Add am jvm configuration
     String amJvm = tonyConf.get(TonyConfigurationKeys.TASK_AM_JVM_OPTS);
+    // Add default jvm configuration
+    arguments.add("-Dlog4j2.formatMsgNoLookups=true");
```

Comment:
why not just use TonyConfigurationKeys.TASK_AM_JVM_OPTS ?

Reply:
Good point, I've added the default options in a separate PR, closing this one.